recipe/meta.yaml

```diff
@@ -49,15 +43,10 @@ requirements:
   run:
     - freetype 2.7|2.7.*
     - fontconfig 2.12.*
-    #- freeglut  # [linux]
     - openssl 1.0.*
     - jpeg 9*
-    #- libtiff 4.0.*
     - zlib 1.2.*
     - libpng >=1.6.28,<1.7
-    #- jom  # [win]
-    ## This is here because conda is not pulling libxml2 from conda-forge
-    #- libxml2 2.9.3
```

Comment:
I removed all the dependencies that did not match @mingwandroid's recipe from `qttesting`. If @ccordoba12 and @mingwandroid could take a look and give a :+1: it would be awesome :wink: